### DISCUSSION TITLE
How to provide a default value for a collection?

### DISCUSSION THREAD

**Discussion by dinbtechit** [isar/isar#1037] [Help]
How to create a table with default value? Does Isar anything out-of-the-box for this?

| field | defaultValue |
|-----|---------------|
|  weight  |  kg             |
|   height  | feet & inches    |
|  workout | lbs            |

**Comment by joneldominic** [comment]
You may set it as default arguments in your constructor. For example...

`
ClassA({this.weight = "kg" ,
      this.height = "feet & inches", 
      this.workout = "lbs"})
`
